<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>306</width>
    <height>340</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hesap Makinesi</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_num_1">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>71</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>1st Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_num_2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>60</y>
      <width>71</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>2nd Number:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNum1">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>45</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditNum2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>75</y>
      <width>120</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonSubstract">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Substract</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonMultiply">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>190</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Multiply</string>
    </property>
   </widget>
   <widget class="QPushButton" name="buttonDivide">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>120</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Divide</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>250</y>
      <width>60</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Result :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelResult2">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>270</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="resultLabel2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>306</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
